src/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj

```diff
@@ -22,6 +22,20 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap10.0.16299-Windows_NT-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap10.0.16299-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap10.0.16299aot-Windows_NT-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap10.0.16299aot-Windows_NT-Release|AnyCPU'" />
+  <!-- Setting PackageDestination is required for validation to succeed since we are harvesting an aot netcore50 asset which we don't want to apply
+  for UAP RS3 so instead we use the netstandard implementation -->
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
+    <PackageDestination Include="lib/netstandard2.0">
+      <TargetFramework>netstandard2.0</TargetFramework>
+    </PackageDestination>
+    <PackageDestination Include="runtimes/aot/lib/uap10.0.16299">
```

Comment:
Do we really want the PNSE assets for this? Didn't we have an older uap or netcore50 asset that we should instead be harvesting?

Reply:
After reading your comment why don't we want the netcore50 aot asset to apply? It is a facade on top of System.Private.Interop which expect should work.

Reply:
Actually I forgot to remove that, I ended up crosscompiling DispatchProxy so this will no longer be required.